src/picker/components/Picker/Picker.js

```diff
@@ -271,7 +271,7 @@ $: {
     if (!databaseLoaded) {
       currentEmojis = []
       searchMode = false
-    } else if (searchText.length >= MIN_SEARCH_TEXT_LENGTH) {
```

Comment:
`MIN_SEARCH_TEXT_LENGTH` is actually used in another place, so if we change from `>=` to `>` it would also need to be updated there:

https://github.com/nolanlawson/emoji-picker-element/blob/b2beb27b2b0119a34e3c78a164e3f59159e4c5c7/src/database/utils/normalizeTokens.js#L12

To me, though, "minimum" implies `>=` rather than `>`, so it would make the most sense for `MIN_SEARCH_TEXT_LENGTH` to be 1.